dev-lang/fpc/fpc_bin-3.0.2.recipe

```diff
@@ -0,0 +1,111 @@
+SUMMARY="Pre-built binaries for the FPC build"
+DESCRIPTION="Pre-built FPC binaries for Haiku i386."
```

Comment:
Should be "Free Pascal is a 32, 64 and 16 bit professional Pascal compiler. It can target many processor architectures: Intel x86 (including 8086), AMD64/x86-64, PowerPC, PowerPC64, SPARC, ARM, AArch64, MIPS and the JVM. Supported operating systems include Linux, FreeBSD, Haiku, Mac OS X/iOS/iPhoneSimulator/Darwin, DOS (16 and 32 bit), Win32, Win64, WinCE, OS/2, MorphOS, Nintendo GBA, Nintendo DS, Nintendo Wii, Android, AIX and AROS. Additionally, support for the Motorola 68k architecture is available in the development versions."

Reply:
OK